docs/introduction/sla.md

```diff
@@ -0,0 +1,208 @@
+
+1. Ensuring normal levels of security
+2. Restoring normal service operation
+3. Restoring customer workloads where possible
+
+Response and resolution times are provided as service level targets (see section 6).
+
+### Fulfilment of service requests
+
+In addition to resolving incidents, the following standard service requests are
+defined and will be fulfilled through the defined support channels:
+
+ * Issues regarding CSC’s identity management service
+ * Issues using this service’s web interface
+ * Issues using the provided APIs
+ * Issues deploying workloads
+ * Issues using workload templates
+
+Response and fulfilment times are provided as service level targets (see section 6).
+
+## Service level targets
+
+The Rahti service level targets adhere to JHS1745 as follows:
+
+Service level targets ||
+--- | ---
+Service Level | B (normal)
+Service time / incident handling | P2 Weekdays 7.00-19.00
+Availability | K2 99%
+Response | V2 reaction: 2h, solution: 1 BD
+
+Availability is calculated by subtracting the service break time from the ideal
+availability during service time. This information is obtained from internal
+monitoring systems and defined as Customers’ ability to manage their workloads,
+and the availability of the workloads.
+
+The Service Provider commits to inform the customer if this SLA is violated or
+a violation is anticipated. For this, email as a communication channel will be
+used. 
+
+A Customer contacts the CSC Service Desk for the case of a possible SLA
+violation. The case will be analysed internally and, if the violation is
+confirmed, CSC will inform the Customer about the reasons for the violation,
+planned mitigation actions and expected resolution time.
+
+## Limitations and constraints
+
+The provisioning of the service under the agreed service level targets is subject to the following limitations and constraints: 
+
+ * Support is provided primarily in the following languages: English, Finnish
+ * Downtimes caused due to upgrades for fixing critical security issues are not considered SLA violations. In the case of critical security upgrades CSC reserves the right to apply the upgrades with minimal notice
+ * A technical failure which affects individual customers does not count as downtime
+ * Failures of single instances of the workload (containers) are expected behaviour, and does not count as downtime
+ * Any other causes outside service provider’s direct control
+
+## Communication, reporting and escalation
+
+### General communication
+
+The following contacts will be generally used for communications related to the
+service in the scope of this SLA:
+
+| Communication channels ||
+--- | ---
+Contact for Customers | servicedesk@csc.fi
+Security incident reporting | (CSC’s Head of Security) security@csc.fi +358 (0) 94 57 2253
+
+### Reporting
+The service availability information is…. The information provided will be limited to service availability and by data security and privacy constraints.
```

Comment:
"The service availability information is…" Seems to be missing some text?

Reply:
I added a statement that the availability report is available through a request to servicedesk